<launch>
  <rosparam command="load" file="$(find grid_map_demos)/config/image_to_gridmap_demo.yaml" />

  <!-- Load the image. -->
  <node pkg="grid_map_demos" type="image_publisher.py" name="image_publisher" output="screen">
    <param name="image_path" value="$(find grid_map_demos)/data/eth_logo.png" />
    <param name="topic" value="~image" />
  </node>

  <!-- Launch the grid map image to grid map demo node. -->
  <node pkg="grid_map_demos" type="image_to_gridmap_demo" name="image_to_gridmap_demo" output="screen" />

  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen" />

  <!-- Launch RViz with the demo configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find grid_map_demos)/rviz/grid_map_demo.rviz" />
</launch>
